index.js

```diff
@@ -186,7 +209,9 @@ class KeyringController extends EventEmitter {
     if (!encryptedVault) {
       throw new Error('Cannot unlock without a previous vault.');
     }
-    await this.encryptor.decrypt(password, encryptedVault);
+
+    const result = await this.encryptor.decrypt(password, encryptedVault);
+    return result;
```

Comment:
jsdoc doesn't suggest this would return anything. Why is that needed?

Reply:
This returns:

```
[
      {
        type: 'HD Key Tree',
        data: {
          mnemonic: [Array],
          numberOfAccounts: 1,
          hdPath: "m/44'/60'/0'/0"
        }
      }
    ]
```

I added for debugging purposes, but yes, it can be removed.